app/models/study.rb

```diff
@@ -3313,12 +3313,13 @@ def set_service_account_permissions
     # only perform check if this is not the default portal project
     if self.firecloud_project != FireCloudClient::PORTAL_NAMESPACE
       begin
+        sa_owner_group = AdminConfiguration.find_or_create_ws_user_group!
         client = FireCloudClient.new(self.user, self.firecloud_project)
-        service_account = Study.firecloud_client.issuer
-        acl = client.create_workspace_acl(service_account, 'OWNER', true, true)
+        group_email = sa_owner_group['groupEmail']
+        acl = client.create_workspace_acl(group_email, 'OWNER', true, false)
```

Comment:
can you explain why we used to set compute_permission to true and now you're moving it to false?

Reply:
In truth, setting it to `true` would never allow the service account to submit workflows.  Since SAs can't get refresh tokens, they can't submit to Cromwell.  Also we never used that permission ever, so I'm just accounting for what has been the de facto state all along.